python/lsst/ts/m2gui/controltab/tab_alarm_warn.py

```diff
@@ -75,10 +89,9 @@ def __init__(self, title, model):
 
         self.widget.setLayout(self._layout)
```

Comment:
I know this isn't part of this commit, but can you please use self.widget() instead and drop widget member from TabDefault? You are duplicating with member parameter functionality Qt already provides. E.g. change all self.widget to self.widget() and TabDefault to:
```python
class TabDefault(QDockWidget):
    """Default table.

    Parameters
    ----------
    title : `str`
        Table's title.
    model : `Model`
        Model class.

    Attributes
    ----------
    widget : `PySide2.QtWidgets.QWidget`
        Widget.
    model : `Model`
        Model class.
    """

    def __init__(self, title, model):
        super().__init__()
        self.setWindowTitle(title)

        self.setWidget(QWidget)

        self.model = model
```

Reply:
Fixed in the commit: 6f1e678. Thanks!